components/checkbox/Checkbox.tsx

```diff
@@ -37,6 +37,8 @@ export interface AbstractCheckboxProps<T> {
   type?: string;
   skipGroup?: boolean;
   required?: boolean;
+  onFocus?: React.FocusEventHandler<HTMLElement>;
+  onBlur?: React.FocusEventHandler<HTMLElement>;
```

Comment:
**rc-checkbox** uses the input tag, wouldn't it be better to specify the more specific type **HTMLInputElement** rather than **HTMLElement**?